tern/analyze/docker/container.py

```diff
@@ -188,7 +188,7 @@ def close_client():
     used after analysis is done'''
     try:
         client.close()
-    except requests.exceptions.ConnectionError:
+    except (AttributeError, requests.exceptions.ConnectionError):
         # it should either already be closed or docker is not setup
         # either way, the socket is closed
```

Comment:
Small request: Please add here that there is no socket in use. Hence we just pass.